quinn-udp/src/cmsg/mod.rs

```diff
@@ -112,6 +112,17 @@ impl<'a, M: MsgHdr> Iterator for Iter<'a, M> {
     fn next(&mut self) -> Option<Self::Item> {
         let current = self.cmsg.take()?;
         self.cmsg = unsafe { self.hdr.cmsg_nxt_hdr(current).as_ref() };
+
+        #[cfg(apple_fast)]
```

Comment:
On non `apple_fast`, is there value in reading a `< std::mem::size_of::<M::ControlMessage>()` control message?

Reply:
Probably not. But there is also no bug (that we know of) on other systems ([as you argued earlier](https://github.com/quinn-rs/quinn/pull/2154#issuecomment-2658548664)). I am fine either way, I just thought keeping the semantics the same is good!

Reply:
:wave: old Max.

I am fine either way. Thanks.

Reply:
Ah, it would have been nice to reproduce this in CI or have some other form of regression test.

Do you think there is much value in adding a regression test that attemps to read a cmsg buffer that ends up returning zerod cmsgs?